spec/lib/grease/adapter_spec.rb

```diff
@@ -13,4 +16,9 @@
 
     it { is_expected.to match a_hash_including(data: "3.14") }
   end
+
+  describe "#inspect" do
+    subject { adapter.inspect }
+    it { is_expected.to match(/#<#{described_class.name}\(#{template_class.name}\):0x[0-9a-f]{14}>/) }
```

Comment:
__[rubocop]__
* Line is too long. [102/100] [:ref](https://github.com/bbatsov/ruby-style-guide#80-character-limits)

<!-- Comment by [SideCI](https://sideci.com) -->